### PR TITLE
Organizing colors

### PR DESCRIPTION
**Pull request by phoenixthefirebird**
organizing the colours in storybook framework

image

it works well so I have decided to keep using the storybook framework. I will be responsible for organizing the components into stories

### CODE REVIEW

**Comment by phoenixthefirebird** [comment]
need to change the default font of the page to HK

File: src/components/colors/Colors.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+colorVisual.splice(-9, 0, 'empty');
+colorVisual.splice(-5, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
+
+const Template = () => <ColorPageContainer>
+  {colorVisual.map((colorName) => (colorName === 'empty') 
+  ? <ColorContainer empty={true}></ColorContainer>
+  : <ColorContainer style={{background: colors[colorName]}} white={Number(colors[colorName][1]) <= 6}>{colorName}</ColorContainer>)}
```
Comment (kevin-zou) [inline src/components/colors/Colors.stories.jsx:45]: If I'm understanding this correctly, we can check if a colour is close enough to white by checking if the first character of its hex code is less than 6?

File: src/components/colors/Colors.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+let colorVisual = Array.from(Object.keys(colors));
+
+colorVisual.splice(-9, 0, 'empty');
+colorVisual.splice(-9, 0, 'empty');
+colorVisual.splice(-5, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
+colorVisual.splice(-3, 0, 'empty');
```
Comment (kevin-zou) [inline src/components/colors/Colors.stories.jsx:40]: Just for my personal understanding, what does this do?

**Review by kevin-zou — APPROVED** [review]
This looks great 🔥 left a few comments just so I can understand it a bit better, but the end result and the rest of the code is good to me!

File: src/components/colors/Colors.stories.jsx (hunk 1 above)
Comment (phoenixthefirebird) [reply src/components/colors/Colors.stories.jsx:45]: it was a change split for the colors we have here:p

File: src/components/colors/Colors.stories.jsx (hunk 2 above)
Comment (phoenixthefirebird) [reply src/components/colors/Colors.stories.jsx:40]: haha it's to keep the color layout we have on Aurora spec! Keeping the some family of colors in one row. Plus I made the empty spots look cool :p

PS: `empty` is rendered as the grey blank divs

File: src/components/colors/Colors.stories.jsx (hunk 1 above)
Comment (kevin-zou) [reply src/components/colors/Colors.stories.jsx:45]: Sorry don't quite get it :') either way this is still good to go through so feel free to merge!

**Pull request merged**